packages/tools/devtools/devtools-example/jest.config.js

```diff
@@ -2,16 +2,24 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+// Get the test port from the global map and set it in env for this test
+const testTools = require("@fluidframework/test-tools");
+const { name } = require("./package.json");
+
+const mappedPort = testTools.getTestPort(name);
+process.env["PORT"] = mappedPort;
 
 module.exports = {
-	preset: "ts-jest",
-	roots: ["<rootDir>/src"],
+	preset: "jest-puppeteer",
+	globals: {
+		PATH: `http://localhost:${mappedPort}`,
+	},
+	testMatch: ["**/ui-smoke-test/?(*.)+(spec|test).[t]s"],
 	transform: {
-		"^.+\\.tsx?$": "ts-jest",
+		"^.+\\.test.ts?$": "ts-jest",
 	},
```

Comment:
I think this is making the existing unit test in `src/test/` to stop running...?

Reply:
Fixed